### PR TITLE
Update the README as one introduction of Hello Project

### PR DESCRIPTION
**Pull request by running-cara**
The README file can be considered as one manual doc for github beginners.
It is just translated based on the github offical doc:
https://guides.github.com/activities/hello-world/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,83 @@
-# hello-world
-Github Hello World项目： 10分钟Github入门
+[TOC]
```
Comment (testcara) [inline README.md]: It cannot be shown correctly! Please make sure it can work well!

Comment (testcara) [inline README.md]: It cannot be shown correctly! Please make sure it can work well!

**Review by testcara — COMMENTED** [review]
Seems comments in line

File: README.md (hunk 1 above)
Comment (testcara) [reply README.md]: I would fix it in the later patch, thank you for pointing it out.

**Comment by testcara** [comment]
It is ok to remove it directly.

**Review by testcara — APPROVED** [review]
+1 No more comments! Thank you!

**Comment by running-cara** [comment]
Thank you for the interview. Now merge it now!

**Pull request merged**